python/src/opendp/polars.py

```diff
@@ -1,3 +1,25 @@
+'''
+The ``opendp.polars`` module adds differential privacy to the `Polars <https://docs.pola.rs>`_ DataFrame library.
+If both ``opendp`` and ``polars`` have been imported,
+these methods are registered under the ``dp`` namespace in `Polars expressions <https://docs.pola.rs/py-polars/html/reference/expressions/index.html>`_:
+
+>>> import polars as pl # Import order does not matter.
+>>> import opendp
+>>> expression = pl.col('numbers').dp.sum((0, 10), scale=1.0)
+>>> print(expression) # doctest: +ELLIPSIS
+col("numbers").clip([0, 10]).sum()...:noise()
+
+Note that ``sum`` is a shortcut which actually implies several operations:
+
+* Cliping the values
+* Summing them
+* Applying Laplace noise to the sum
+
+An expression like this can be used as a plan in :py:func:`opendp.measurements.make_private_lazyframe`;
+See the full example there for more information.
+'''
```

Comment:
This is a much better docstring than the pithy one-liner I have in Context API integration. Thanks!